src/playbacks/html5_video/html5_video.js

```diff
@@ -494,25 +495,33 @@ export default class HTML5Video extends Playback {
 
   setClosedCaptionsTrackId(trackId) {
     let tracks = this.closedCaptionsTracks
+    // Since it is possible to display multiple tracks,
+    // ensure that all tracks are hidden.
+    let showingTrack
     for (let i = 0; i < tracks.length; i++) {
       let track = tracks[i]
+      // Check if track to show
       if (trackId === track.id) {
-        if (track.track.mode === 'hidden' || track.track.mode === 'disabled') {
-          track.track.mode = 'showing'
-          this._ccTrackId = trackId
-        }
-      } else if (track.track.mode === 'showing') {
+        showingTrack = track
+      }
+      if (track.track.mode !== 'hidden') {
+        // In theory 'disabled' may be more optimized, but
+        // it leads to weird behaviours with some browsers.
+        // FIXME: value set could be browser specific ?
         track.track.mode = 'hidden'
       }
     }
```

Comment:
thx @leandromoreira , i will apply these changes. Note also that trackId can equals -1 (_to disable cc_).